src/core/config.ts

```
@@ -4,8 +4,89 @@
+    try {
+      // First approach: Try to convert to valid JSON by wrapping in curly braces
+      const jsonString = `{${landoInfo.replace(/'/g, '"')}}`;
+      try {
+        const info = JSON.parse(jsonString);
+        Printer.log(`Parsed Lando info successfully`);
+
+        // Look for the database service
+        const databaseService = info.find(
+          (service: { service: string }) => service.service === 'database'
+        );
+        if (databaseService && databaseService.external_connection) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)